Makefile.toml

```
@@ -259,11 +260,20 @@ chmod -R o+r ${CARGO_HOME}
 dependencies = ["fetch-sdk"]
 script = [
 '''
-${BUILDSYS_TOOLS_DIR}/docker-go \
-  --module-path ${BUILDSYS_SOURCES_DIR}/host-ctr \
-  --sdk-image ${BUILDSYS_SDK_IMAGE} \
-  --go-mod-cache ${GO_MOD_CACHE} \
-  --command "go list -mod=readonly ./... >/dev/null && go mod vendor"
+go_fetch()
+{
+  local module
+  module="${1:?}"
+  ${BUILDSYS_TOOLS_DIR}/docker-go \
+    --module-path "${BUILDSYS_SOURCES_DIR}/${module}" \
+    --sdk-image ${BUILDSYS_SDK_IMAGE} \
+    --go-mod-cache ${GO_MOD_CACHE} \
+    --command "go list -mod=readonly ./... >/dev/null && go mod vendor"
+}
+
+for m in ${GO_MODULES}; do
+  go_fetch $m
```

nit:
```suggestion
  go_fetch ${m}
```

(same for other places where we're using the $m or $module variables)